docs/serverless/03-03-registries.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Internal and external registries
+type: Details
+---
+
+In its default configuration, Serverless uses persistent volumes as the internal registry to store Docker images for Functions. The default storage size of a single volume is 20 GB. This internal registry is suitable for local development.
+
+If you use Serverless for production purposes, it is recommended that you use an external registry, such as Docker Hub, Google Container Registry (GCR), or Azure Container Registry (ACR).
+
+Serverless supports two ways of connecting to an external registry:
+
+- [You can set up an external registry before installation](#tutorials-set-an-external-docker-registry).
+
+  In this scenario, you can use Kyma overrides to change the default values supplied by the installation mechanism.
+
+- [You can switch to an external registry in the runtime](#tutorials-switch-to-an-external-docker-registry).
+
+  In this scenario, you can change the registry on the fly, with Kyma already installed on your cluster. This option gives you way more flexibility and control over the choice of an external registry for your Functions.
+
+## Switching registries in the runtime
+
+When you install Kyma with the default internal registry, Helm creates the `serverless-registry-config-default` Secret in the `kyma-system` Namespace. This Secret contains credentials used by Kubernetes to pull deployed Functions' images from the internal registry. They also allow Kaniko to push any images to the registry each time a Function is created or updated.
+
+> **NOTE:** If you [install Serverless with overrides](#tutorials-set-an-external-docker-registry), disabling the internal registry and specifying an external one to use, the `serverless-registry-config-default` Secret will contain credentials to access the specified external registry instead.
```

Comment:
```suggestion
> **NOTE:** If you [install Serverless with overrides](#tutorials-set-an-external-docker-registry), disable the internal registry and specify an external one to use, the `serverless-registry-config-default` Secret will contain credentials to access the specified external registry instead.
```
That's just my preference, feel free to ignore 😺

Reply:
I modified it in this way (to better explain that installing with overrides means disabling the default internal registry):
```> **NOTE:** If you [install Serverless with overrides](#tutorials-set-an-external-docker-registry), you disable the internal registry and specify an external one to use. The `serverless-registry-config-default` Secret will then contain credentials to the specified external registry instead of the internal one.```